main.py

```diff
@@ -220,5 +233,7 @@ def single_argument_handler(arg):
               "\nOr type \"help\" to see all the commands.")
     elif len(argv) == 2:
         single_argument_handler(argv[1].lower())
+    elif argv[1].lower().startswith("slideshow"):
```

Comment:
What if the user types in 3 parameters or more like this:
```$ pokemon slideshow 0.25 abc 123 def```
Even though this will still technically work, its best to let the user know that they are using the command incorrectly.
It would be better practice to be more strict about this:
```elif len(argv) == 3:```
Then check for "slideshow" inside the method.